go.mod

```diff
@@ -1,16 +1,14 @@
 module github.com/prometheus/pushgateway
 
-go 1.23
-
-toolchain go1.23.0
+go 1.23.0
 
 require (
 	github.com/alecthomas/kingpin/v2 v2.4.0
 	github.com/golang/protobuf v1.5.4
-	github.com/golang/snappy v0.0.4
-	github.com/prometheus/client_golang v1.21.0
+	github.com/golang/snappy v1.0.0
```

Comment:
Side note, it looks like this library is now archived.

Reply:
Weird. They cut v1.0.0 just two weeks ago and then archived the repo, without any messaging around it in the Godoc or the Readme. Maybe the archival was accidental?